server/api/card.js

```diff
@@ -90,14 +92,18 @@ router.post('/make25/forRoom/:roomId', async (req, res, next) => {
       where: { name: roomId },
     });
 
+    if (!room) return res.sendStatus(404); // Sanity check
+
     room.setBoard(board);
+    console.log('hit here');
```

Comment:
These console logs are useful for testing but since this branch has conflicts Im gonna mark the ones I think can be taken out once the merge conflict is resolved and the review is done!

Reply:
I sometimes get a 500 when starting up the game and was trying to figure out where it broke, sorry! I still can't figure out when this occurs / how to this happens...